includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -922,6 +931,9 @@ private function get_stylesheet_priority( DOMNode $node ) {
 				'wp-mediaelement',
 				'thickbox',
 			];
+			$low_priority_plugins  = [
+				'query-monitor',
```

Comment:
Any suggestions of what else to add here?

Reply:
None that I can think of at the moment.

Reply:
What about making this list of plugins a sanitizer arg, with `[ 'query-monitor' ]` being the initial value? So instead of `$low_priority_plugins` here it could use `$this->args['low_priority_plugins']`. This would allow the plugins to be configurable via the `amp_content_sanitizers` filter.

Also: Query Monitor should ideally be short-lived here since we want to add dev mode support directly to Query Monitor.

Reply:
Via https://github.com/ampproject/amp-wp/pull/3877/commits/518930d9adbeab05546921120f91ce4cadae51d2